.github/pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+<!--
+    Thank you for your interest in contributing to Nautobot ChatOps App! Please note
+    that our contribution policy recommends that a feature request or bug
+    report be opened for approval prior to filing a pull request. This
+    helps avoid wasting time and effort on something that we might not
+    be able to accept.
+
+    Please indicate the relevant feature request or bug report below.
+-->
+
+# Closes: #<ISSUE NUMBER GOES HERE>
+
+## What's Changed
+
+<!--
+    Please include:
+    - A summary of the proposed changes
+    - A sectioned breakdown for larger features under ## subheadings
+    - Screenshots, example payloads where relevant:
+      - Before/After for bugfixes
+      - Using a new feature
+-->
+
+## To Do
+
+<!--
+    Please feel free to update todos to keep track of your own notes for WIP PRs.
+-->
+- [ ] Explanation of Change(s)
+- [ ] Added change log fragment(s) (for more information see [the documentation](https://docs.nautobot.com/projects/core/en/stable/development/#creating-changelog-fragments))
```

Comment:
We need to get an issue opened to fix this link if we don't already have one. It should be

```suggestion
- [ ] Added change log fragment(s) (for more information see [the documentation](https://docs.nautobot.com/projects/core/en/stable/development/core/#creating-changelog-fragments))
```

Reply:
I just tacked it on as a comment [here](https://github.com/nautobot/cookiecutter-nautobot-app/issues/177#issuecomment-2608461714)